### PR TITLE
ApiManagement - renaming /properties entity to /namedValues

### PR DESCRIPTION
**Pull request by vfedonkin**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimanagement.json [hunk 1]
```diff
@@ -530,12 +530,12 @@
       "maxLength": 256,
       "x-ms-parameter-location": "method"
     },
-    "PropertyIdParameter": {
+    "NamedValueIdParameter": {
       "name": "propId",
```
Comment (solankisamir) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimanagement.json]: >propId [](start = 15, length = 6)

rename to nameValueId

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimnamedvalues.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+    "description": "Use these REST APIs for performing operations on NamedValue entity associated with your Azure API Management deployment. API Management policies are a powerful capability of the system that allow the publisher to change the behavior of the API through configuration. Policies are a collection of statements that are executed sequentially on the request or response of an API. Policy statements can be constructed using literal text values, policy expressions, and NamedValues. Each API Management service instance has a NamedValues collection of key/value pairs that are global to the service instance. These NamedValues can be used to manage constant string values across all API configuration and policies.",
```
Comment (solankisamir) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimnamedvalues.json]: lowercase

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimnamedvalues.json [hunk 3]
```diff
@@ -34,19 +34,19 @@
...
         "tags": [
-          "Property"
+          "NamedValue"
         ],
-        "operationId": "Property_ListByService",
-        "description": "Lists a collection of properties defined within a service instance.",
+        "operationId": "NamedValue_ListByService",
+        "description": "Lists a collection of NamedValues defined within a service instance.",
         "externalDocs": {
           "url": "https://docs.microsoft.com/en-us/azure/api-management/api-management-howto-properties"
```
Comment (solankisamir) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimnamedvalues.json]: >properties [](start = 93, length = 10)

we should ask the PM to rename the doc name :)

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimanagement.json (hunk 1 above)
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimanagement.json]: once you do that.. you will need to change the examples also.

---
In reply to: [341407943](https://github.com/Azure/azure-rest-api-specs/pull/7683#discussion_r341407943) [](ancestors = 341407943)

**Review by solankisamir — CHANGES_REQUESTED** [review]
left some feedback. rest looks good.

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimnamedvalues.json (hunk 2 above)
Comment (vfedonkin) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimnamedvalues.json]: Usually we use upper case. Like "Lists a collection of all external Caches in the specified service instance."

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimanagement.json (hunk 1 above)
Comment (vfedonkin) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimanagement.json]: Done

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimnamedvalues.json (hunk 3 above)
Comment (vfedonkin) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimnamedvalues.json]: I asked Mike to do this

**Review by solankisamir — APPROVED** [review]
:shipit:

**Pull request merged**